phylogenetic/rules/export.smk

```diff
@@ -83,6 +83,21 @@ rule prepare_auspice_config:
                 "title": "Dengue Genotype (Nextclade)",
                 "type": "categorical"
               },
+              {
+                "key": "genotype",
+                "title": "Genotype (Nextclade)",
+                "type": "categorical"
+              },
+              {
+                "key": "major_lineage",
+                "title": "Major lineage (Nextclade)",
+                "type": "categorical"
+              },
+              {
+                "key": "minor_lineage",
+                "title": "Minor lineage (Nextclade)",
+                "type": "categorical"
+              },
```

Comment:
_non-blocking suggestion_

Maybe just because I'm not regularly working with dengue, but it's not clear to me what the difference is with all the coloring options just by the title, e.g. what's the difference between "Dengue Genotype (Nextclade)" and "Genotype (Nextclade)"? 

By my count, there are 6(!) different coloring options for serotype/genotype/lineage. I think it would be helpful to add a brief description of each in the description.md.

Reply:
Ha, it is confusing. I wrote out a description of the different options here, but feel free to suggest further edits: https://github.com/nextstrain/dengue/pull/93/commits/b91c485791d31cb084c974053889b5d43d03f6e7